### ISSUE TITLE
Add link text to logo header

### ISSUE THREAD

**Issue by cstarcher** [TexasDigitalLibrary/Vireo#1659]
Description
The website logo in the header, a link to the “Thesis Submission Portal” home page, is missing link text.

People with vision disabilities who use screen readers rely on alt text to understand the data displayed in an image. Without alt text, the screen reader automatically announces the file name or path, which can lead to confusion.

Adding an {{alt}} attribute with a value describing the link destination will allow users with visual disabilities to understand where the link will take them.

Note: Avoid using “picture of” or “Logo” in the {{alt}} text. The screen reader will announce that the component is an image. Repeating information can be confusing for any user.

Current Code
```
<!-- VIREO Logo -->
<a class="pull-left" href="/">
    <img style="max-height: 57px;" ng-src="resources/images/default-left-logo.png" src="resources/images/default-left-logo.png">
</a>

<!-- Thesis and Disertation Management System Logo-->
<a class="pull-left" href="/">
    <img style="max-height: 57px;" ng-src="resources/images/logo.png" src="resources/images/logo.png">
</a>
```

Solution
Add an alt attribute with a value describing the link destination (“Thesis Submission Portal Home”) to allow users with visual disabilities to understand where the link will take them.
Recommended Code
```
<!-- VIREO Logo -->
<a class="pull-left" href="/">
    <img alt="Thesis Submission Portal Home" style="max-height: 57px;" ng-src="resources/images/default-left-logo.png" src="resources/images/default-left-logo.png">
</a>

<!-- Thesis and Disertation Management System Logo -->
<!-- Note that this link takes the user to the "Thesis Submission Portal" 
homepage, the same page that the VIREO logo link goes to. The logo should 
be the same on all pages to avoid confusion. -->
<a class="pull-left" href="/">
    <img alt="Thesis Submission Portal Home" style="max-height: 57px;" ng-src="resources/images/default-left-logo.png" src="resources/images/default-left-logo.png">
</a>
```
